README.md

```diff
@@ -1,53 +1,69 @@
 
-**Note**: By running the code in this action, you will be executing third-party
-code that is licensed under separate terms.
+### Run MATLAB Script on GitHub-Hosted Runner
+Set up a GitHub-hosted runner to run the commands in a file named `myscript.m` in the root of your repository. To run the script, include the [Run MATLAB Command](https://github.com/matlab-actions/run-command/) action in your workflow.
 
-## Usage
-
-You can use this action `with`:
-| Argument  | Description |
-|-----------|-------------|
-| `release` | (Optional) MATLAB release to install. You can specify R2020a or a later release. By default, the action installs the latest release of MATLAB. <br/> **Example**: `R2020a` <br/> **Default**: `latest`
+```yaml
+name: Run MATLAB Script on GitHub-Hosted Runner
+on: [push]
+jobs:
+  my-job:
+    name: Run MATLAB Script
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Install MATLAB
+        uses: matlab-actions/setup-matlab@v0
+      - name: Run script
+        uses: matlab-actions/run-command@v0
+        with:
+          command: 'myscript'
+```
 
-## Example
+### Run MATLAB Tests on GitHub-Hosted Runner
+Set up a GitHub-hosted runner to automatically run the tests in your [MATLAB project](https://www.mathworks.com/help/matlab/projects.html) and generate a JUnit test results report and a Cobertura code coverage report. To run the tests and generate the artifacts, include the [Run MATLAB Tests](https://github.com/matlab-actions/run-tests/) action in your workflow.
 
 ```yaml
-name: Sample workflow
+name: Run MATLAB Tests on GitHub-Hosted Runner
 on: [push]
-
 jobs:
   my-job:
-    name: Say hello from MATLAB
+    name: Run MATLAB Tests and Generate Artifacts
     runs-on: ubuntu-latest
     steps:
-      - uses: matlab-actions/setup-matlab@v0
-      - name: Run MATLAB from the system shell
-        run: matlab -batch "disp('hello world')"
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Install MATLAB
+        uses: matlab-actions/setup-matlab@v0
+      - name: Run tests and generate artifacts
+        uses: matlab-actions/run-tests@v0
+        with:
+          test-results-junit: test-results/results.xml
+          code-coverage-cobertura: code-coverage/coverage.xml
 ```
 
-You can use this action with the [Run MATLAB Command](https://github.com/matlab-actions/run-command/) and [Run MATLAB Tests](https://github.com/matlab-actions/run-tests/) actions to easily run MATLAB and Simulink as part of your build pipeline.
+## Set Up MATLAB
+When you define your workflow in the `.github/workflows` directory of your repositoy, you can specify the **Set Up MATLAB** action using the `setup-matlab` key. The action accepts an optional input.
```

Comment:
Like the other two, `setup-matlab` prolly won't be used unqualified without our org/version 🤔

Reply:
Updated the text :).